src/Sdk/StrixMusic.Sdk.Tests/Mock/Core/MockCore.cs

```diff
@@ -57,7 +57,10 @@ public async Task InitAsync(CancellationToken cancellationToken = default)
 
         public event EventHandler<string>? InstanceDescriptorChanged;
 
-        public CoreMetadata Registration { get; } = new CoreMetadata(nameof(MockCore), "Mock core", new Uri("https://strixmusic.com/"), Version.Parse("0.0.0.0"));
+        public CoreMetadata Registration { get; } = new CoreMetadata(id: nameof(MockCore),
+                                                                     displayName: "Mock core",
+                                                                     logoUri: new Uri("https://strixmusic.com/"),
+                                                                     sdkVer: typeof(ICore).Assembly.GetName().Version!);
```

Comment:
Rather than suppressing nullables here, we should throw
```suggestion
                                                                     sdkVer: typeof(ICore).Assembly.GetName().Version ?? throw new ArgumentNullException());
```